tests/SbrpTests/SbrpTests.Tests.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="NuGet.Packaging" Version="$(NuGetPackagingVersion)" />
+    <PackageReference Include="Xunit.SkippableFact" Version="1.4.13" />
```

Comment:
Consider moving that property into Versions.props. That's what we do in other repos and that will make switching to NuGet Central Package Management in the future easier.

Reply:
Thanks for the comment, I've moved the property into Versions.props in https://github.com/dotnet/source-build-reference-packages/pull/817/commits/417c9953c921b03eecb143ae7fb4dec5a5fc4fcc